### PR TITLE
BL-2246 Don't publish PDF files except main preview

### PR DESCRIPTION
**Pull request by JohnThomson**
Prevents wasting space and bandwidth on any
obsolete previews or other PDF junk that has crept
into the book folder

### CODE REVIEW

**Comment by gmartin7** [comment]
How does the user tell UploadBook which file is the right preview pdf file?

**Comment by JohnThomson** [comment]
The user doesn't; it's hard-coded; desktop and website code agree on how to derive the name (I think it's just the book name.pdf).

File: src/BloomExe/WebLibraryIntegration/BloomS3Client.cs [hunk 1]
```diff
@@ -158,7 +159,11 @@ public void UploadBook(string storageKeyOfBookFolder, string pathToBloomBookDire
 
 			Directory.CreateDirectory(wrapperPath);
 
-			CopyDirectory(pathToBloomBookDirectory, Path.Combine(wrapperPath, Path.GetFileName(pathToBloomBookDirectory)));
+			var destDirName = Path.Combine(wrapperPath, Path.GetFileName(pathToBloomBookDirectory));
+			CopyDirectory(pathToBloomBookDirectory, destDirName);
+			var unwantedPdfs = Directory.EnumerateFiles(destDirName, "*.pdf").Where(x => Path.GetFileName(x) != pdfToInclude);
```
Comment (hatton) [inline src/BloomExe/WebLibraryIntegration/BloomS3Client.cs]: John, should we add "thumbs.db" to that?

Comment (JohnThomson) [inline src/BloomExe/WebLibraryIntegration/BloomS3Client.cs]: The unit test seems to indicate that something else is already preventing upload of thumbs.db, though I don't know what.

Comment (hatton) [inline src/BloomExe/WebLibraryIntegration/BloomS3Client.cs]: OK, perhaps it is a "hidden" file and something is already skipping it.

**Pull request merged**